utils/fileUtils.go

```diff
@@ -1,16 +1,21 @@
 package utils
 
 import (
+	"encoding/json"
 	"fmt"
 	"github.com/apex/log"
+	"io"
+	"io/fs"
+	"io/ioutil"
 	"os"
 	"path/filepath"
+	"strings"
 )
 
 // Searches recursively for full path(es) of a given filename. Returns a 1-elem
 // array if there is just one occurrence, or an array with several elements otherwise.
-func FindFiles(justTheFileName, rootDir string) ([]string, error) {
-	log.Debug("utils.FindFiles() - justTheFileName: " + justTheFileName + ", rootDir: " + rootDir)
+func FindFilesInDirRecursively(justTheFileName, rootDir string) ([]string, error) {
+	log.Debug("utils.FindFilesInDirRecursively() - justTheFileName: " + justTheFileName + ", rootDir: " + rootDir)
 	var targetPaths []string
 
 	err := filepath.Walk(rootDir, func(currentPath string, info os.FileInfo, err error) error {
```

Comment:
I will test the Glob variant instead. 

However, I disagree with the 2nd comment. Since I have done a couple of patches already, I know what I want: I want to type just the filename, without worrying about the path, or dragging/dropping it. (If I want to drag/drop, I can still do so.)

Reply:
Undo the 1st change as well. Glob doesn't support "**".